CHANGELOG.rst

```diff
@@ -1,5 +1,52 @@
 Changelog
 =========
+1.8.0 (2020-04-16)
+------------------
+MLflow 1.8.0 includes several major features and improvements:
+
+Features:
+
+- Added new APIs for Azure Deployment (#2375 @csteegz, #2711, @akshaya-a)
+- Added support for case sensitive LIKE and ILIKE operator in SQLite (#2708, @mparkhe)
+- Added line smoothing on Exponential Moving Average (#2620, @Valentyn1997)
+- Adding support for spacy flavor (#2242, @arocketman)
+- Separating selection checkbox in mlflow UI to have it's own column (#2559, @harupy)
+- Highlighting different parameter when comparing runs (#2565, @gabrielbretschner)
+- Add run comparison in model registry (#2537, @juntai-zheng)
+- Adding UI support for non-numerical experiment ID's (#2568, @jonas)
+- Docker run now accepts arguments in CLI (#2608, @ksanjeevan)
+- Adding windows support for pushing images to aws ecr (#2500, @AndreyBulezyuk)
+
+Bug fixes:
+
+- Fixed CLI summary message in mlflow azureml build_image CLI (#2712, @dbczumar)
+- Updated flower_classifier/score_images_rest.py with multiple bug fixes (#2647, @tfurmston)
+- Fixed pip not found error while packaging models via mlflow models build-docker (#2699, @HiromuHota)
+- Fixed bug causing erroneous deletion of TensorBoard logging directory (#2670, @dbczumar)
+- Fixed a bug that truncated the description of the mlflow gc subcommand in mlflow --help (#2679, @dbczumar)
+- Fixed bug where Travis builds and MLFlow Docker packaging was failing due to incorrect Miniconda download URL (#2685, @michaeltinsley)
```

Comment:
```suggestion
- Fixed bug where ``mlflow models build-docker`` was failing due to incorrect Miniconda download URL (#2685, @michaeltinsley)
```

Don't need to mention the Travis fix here as it's not relevant to users

Reply:
Done.